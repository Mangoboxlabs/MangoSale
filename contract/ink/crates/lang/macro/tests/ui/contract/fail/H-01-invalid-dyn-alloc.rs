use ink_lang as ink;

#[ink::contract(dynamic_storage_allocator = "foo")]
mod invalid_version {
    #[ink(storage)]
    pub struct InvalidVersion {}

    impl InvalidVersion {
        #[ink(constructor)]
        pub fn constructor() -> Self {
            Self {}
        }

        #[ink(message)]
        pub fn message(&self) {}
    }
}

fn main() {}
