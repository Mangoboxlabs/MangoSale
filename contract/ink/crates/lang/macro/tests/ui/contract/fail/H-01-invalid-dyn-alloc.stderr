error: expected a bool literal for `dynamic_storage_allocator` ink! configuration argument
 --> $DIR/H-01-invalid-dyn-alloc.rs:3:17
  |
3 | #[ink::contract(dynamic_storage_allocator = "foo")]
  |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
